[audit] Add severity level information for conan audit scan (#4080)
conan-io/docs: devops/audit.rst

    :align: center
    :alt: Conan audit report

The scan also has the threshold option ``--severity-level``, which allows you to set a minimum severity level for the vulnerabilities.

> In case the threshold value is lower than of the vulnerabilities found, the command will return a non-zero exit code.

Reviewer: ```suggestion
In case the threshold value is surpassed by any of the vulnerabilities found, the command will return a non-zero exit code.
```
